GDJS/Runtime/inputmanager.ts

```diff
@@ -295,6 +304,14 @@ namespace gdjs {
       return this._touches.get(identifier).y;
     }
 
+    /**
+     * @param identifier the touch identifier
+     * @returns true if the touch has just ended.
+     */
+    hasTouchEnded(identifier: integer): boolean {
```

Comment:
nit/idea: we could rename all `identifier` parameters to `rawIdentifier` (coming from the browser API) or `publicIdentifier` (the one stored by GDevelop)